### PR TITLE
feat: add lockupUserBalance

### PR DESCRIPTION
**Pull request by yoojinko**
Add new method lockupUserBalance

### CODE REVIEW

**Review by woomurf — APPROVED** [review]
LGTM, Thanks!

File: src/asset.ts [hunk 1]
```diff
@@ -230,4 +230,23 @@ export default class Asset extends AinftBase {
     const trailingUrl = `credit/${symbol}/withdraw/complete`;
     return this.sendRequest(HttpMethod.POST, trailingUrl, body);
   }
+
```
Comment (woomurf) [inline src/asset.ts:233]: minor) Could you add description as jsDoc?

Comment (yoojinko) [reply src/asset.ts]: I added description lockupUserBalance!
I will soon add a description of all the functions I added.

**Comment by yoojinko** [comment]
Thanks for the review!

**Pull request merged**